spec/policies/exercise_policy_spec.rb

```diff
@@ -186,26 +186,6 @@
     end
   end
 
-  permissions :submit? do
```

Comment:
Deleting this policy is okay, but we should adjust the specs for the `SubmissionPolicy` to test for the newly-introduced `finalize?` as well.